service.subtitles.napiprojekt/addon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.napiprojekt" name="Napiprojekt.pl" version="1.2.0" provider-name="CaTz, Kofii">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.requests"/>
+        <import addon="script.module.future"/>
+    </requires>
+    <extension point="xbmc.subtitle.module" library="service.py"/>
+    <extension point="xbmc.addon.metadata">
+		<summary lang="en">Napiprojekt.pl</summary>
+		<description lang="en_GB">Search and Download subtitles from napiprojekt.pl.</description>
+		<source>https://github.com/mkaflowski/repo-scripts</source>
```

Comment:
this is not the upstream location of your code, this is your fork of repo-scripts...